src/Misc/layoutbin/darwin.svc.sh.template

```diff
@@ -40,6 +42,14 @@ function install()
     cp "${TEMP_PATH}" "${PLIST_PATH}" || "failed to copy plist"
     cp ./bin/runsvc.sh ./runsvc.sh
     chmod 755 ./runsvc.sh
+
+    echo '
+    {
+        "RunAsService": true,
+        "ServiceName": "'${SVC_NAME}'",
```

Comment:
I forgot we discussed that the file needs only a string. This makes sense and I see you made the change already.